gnomad_hail/resources/grch37/reference_data.py

```diff
@@ -100,90 +149,114 @@
 
 lcr_intervals = TableResource(
     path="gs://gnomad-public/resources/intervals/LCR.GRCh37_compliant.interval_list.ht",
-    import_sources={
-        "source_path": "gs://gnomad-public/intervals/LCR.GRCh37_compliant.interval_list",
+    import_func=hl.import_locus_intervals,
+    import_args={
+        "path": "gs://gnomad-public/intervals/LCR.GRCh37_compliant.interval_list",
```

Comment:
since we are using `skip_invalid_loci` on vcfs, should we use `skip_invalid_intervals` on bed files/lnterval files?